src/templates/settings.html

```diff
@@ -63,7 +63,7 @@
     </div>
   </section>
 
-  <section class="main full hide-on-desktop account-settings-links">
```

Comment:
This should change to `hide-on-tablet` instead of just removing `hide-on-desktop`.